src/fun/parser.rs

```diff
@@ -1093,6 +1127,15 @@ impl<'a> Parser<'a> for TermParser<'a> {
     self.expected_spanned(exp, ini_idx..end_idx)
   }
 
+  /// Generates an error message with an additional custom message.
+  ///
+  /// Override to have our own error message.
```

Comment:
This one osnt an override

Reply:
It's in the newer version of the TSPL crate